.github/workflows/lektor-server.yml

```diff
@@ -0,0 +1,22 @@
+name: Building and Deploying to CC opensource site
+on: [push]
+jobs:
+ build-and-deploy:
+  runs-on: ubuntu-latest
+  steps:
+   - uses: actions/checkout@v1
+   - name: setup python version
+     uses: actions/setup-python@v1
+     with:
+      python-version: 3.7
+   - name: Install dependencies
+     run: |
+      pip install pipenv
+      pipenv install
+      pipenv run lektor build -f webpack --output-path ./build 
+   - name: Deploy
+     uses: peaceiris/actions-gh-pages@v3
+     with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_dir: ./build
+          publish_branch: gh-pages
```

Comment:
We deploy to a different repo (https://github.com/creativecommons/creativecommons.github.io), we do not use a `gh-pages` branch.

Reply:
Please use the `lektor deploy ghpages` script as shown in the current Travis config: https://github.com/creativecommons/creativecommons.github.io-source/blob/master/.travis.yml

Reply:
Will running the command `lektor deploy ghpages` do all the work? 
I am not sure how I will go about deploying to a different repo? Are there any other commands I have to run along?

Reply:
You don't need to run anything else. The deployment is configured here: https://github.com/creativecommons/creativecommons.github.io-source/blob/master/.lektorproject